core/chains/evm/config/chain_specific_config.go

```diff
@@ -325,6 +325,20 @@ func setChainSpecificConfigDefaultSets() {
 	optimismGoerli := optimismKovan
 	optimismGoerli.linkContractAddress = "0xdc2CC710e42857672E7907CF474a69B63B93089f"
 
+	// Optimism's Bedrock upgrade uses EIP-1559.
+	optimismBedrock := fallbackDefaultSet
+	optimismBedrock.eip1559DynamicFees = true
+	optimismBedrock.blockEmissionIdleWarningThreshold = 30 * time.Second
+	optimismBedrock.nodeDeadAfterNoNewHeadersThreshold = 60 * time.Second
```

Comment:
Is this safe? Does bedrock regularly emit new heads now?

Reply:
Yes, every 2s regardless of whether there are transactions to put in them or not.